spring-integration-mongodb/src/main/java/org/springframework/integration/mongodb/inbound/MongoDbMessageSource.java

```diff
@@ -37,7 +38,7 @@
 import org.springframework.util.Assert;
 import org.springframework.util.CollectionUtils;
 
-import com.mongodb.DBObject;
+import java.util.List;
```

Comment:
This is minor and I'll fix on merge, but the order of imports in Spring Integration is like: https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style#import-statements